snforge_std/src/cheatcodes/events.cairo

```diff
@@ -29,50 +18,71 @@ struct Event {
     data: Array<felt252>
 }
 
-
-/// An event spy structure, along with the events collected so far in the test.
-/// `events` are mutable and can be updated with `fetch_events`.
+/// An event spy structure containing an offset marking from which event given Spy
+/// should fetch events.
```

Comment:
```suggestion
/// An event spy structure allowing to only get events emitted after it was created.
```

Offset is in fact implementation detail for the point in time from which we want to spy